examples/c/Makefile.am

```diff
@@ -27,6 +25,9 @@ noinst_PROGRAMS = \
 	ex_stat \
 	ex_sync \
 	ex_thread
+# Move this to the end as autotools treats everything after # as a comment
+# ex_backup
+# ex_backup_block
```

Comment:
Why are the backup examples are commented out? I don't see anything in these changes that should impact them.

Reply:
A bunch of backup tests are also failing.

Reply:
I think the backup tests are failing because this change exposes the problem that transaction ids not being cleared in some cases in reconciliation and a lot of records become invisible after restart.

Reply:
@agorrod @keithbostic @bvpvamsikrishna I now understand why the backup tests are failing. The base_write_gen in the backup folder starts from 1 instead of the original base_write_gen + 1.

Reply:
@quchenhao  Do you want this to be looked at? Or will you address the issue with `base_write_gen` first?

Reply:
@tetsuo-cpp I will first merge WT-5508, then address the base_write_gen issue. However, feel free to look at this change as I think it is complete except some minor comments from Alex G.